modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -919,7 +919,7 @@ public void testState() throws Exception {
 
         assertEquals(EXIT_CODE_OK, execute("--state"));
 
-        assertContains(log, testOut.toString(), "Cluster is active");
+        assertContains(log, testOut.toString().replaceAll("\r", ""), "Cluster state: ACTIVE\n");
```

Comment:
Why did you use `replaceAll` and "\n"?

Reply:
The *“Cluster state: ACTIVE”* string in this test will work for both “ATCIVE” and “ACTIVE_READ_ONLY”, so specifically for the “ATCIVE” state we need an end of line symbol (“\n”) in the string.

The “\n” symbol however works only for MacOS and Linux, in Windows it’s “\r\n”. So, for this test to work in all operating systems it’s necessary to delete the “\r” from the output string.

Reply:
Let's replace this check with `Pattern` matching then.

Reply:
Pattern/Matcher mechanism for checking output strings is implemented.